base/src/main/java/org/mozilla/jss/pkcs12/PFX.java

```diff
@@ -164,17 +164,8 @@ public boolean verifyAuthSafes(Password password, StringBuffer reason)
                 return false;
             }
 
-        } catch (java.security.DigestException e) {
-            reason.append("A DigestException occurred");
-            return false;
-
-        } catch (TokenException e) {
-            reason.append("A TokenException occurred");
-            return false;
-
-        } catch (CharConversionException e) {
-            reason.append("An exception occurred converting the password from chars to bytes");
-            return false;
+        } catch (Exception e) {
+            throw e;
```

Comment:
Since we are not doing any action with the exception then try/catch could be totally removed.

Reply:
Updated, the try-catch is removed now.